examples/rosetta/AplusB/main.roc

```diff
@@ -0,0 +1,54 @@
+app "rosetta-example"
+    packages { pf: "https://github.com/roc-lang/basic-cli/releases/download/0.3.1/97mY3sUwo433-pcnEQUlMhn-sWiIf_J9bPhcAFZoqY4.tar.br" }
+    imports [
+        pf.Stdout,
+        pf.Task,
+        pf.Arg,
+        Str,
+    ]
+    provides [main] to pf
+
+TaskErrors : [InvalidArg, InvalidNumStr]
+
+main =
+    task =
+        args <- readArgs |> Task.await
+
+        sum = args.a + args.b
+        aStr = args.a |> Num.toStr
+        bStr = args.b |> Num.toStr
+        sumStr = sum |> Num.toStr
+
+        Task.succeed "The sum of \(aStr) and \(bStr) is \(sumStr)"
+
+    taskResult <- Task.attempt task
+
+    when taskResult is
+        Ok result -> Stdout.line result
+        Err InvalidArg -> Stdout.line "Error: Please provide two integers between -1000 and 1000 as arguments."
```

Comment:
Why the arbitrary constraint on number size?

Reply:
Yeah, that was a Rosetta thing.. I'll kill it for the non-Rosetta solution